### DISCUSSION TITLE
Application rejecting based on location

### DISCUSSION THREAD

**Discussion by cache999** [community/community#65254] [GitHub Education]
### Hi everyone,

I recently tried to apply for Github Student Pack but my application wasn't approved partly because of my geographical location. I'm currently studying at a university in the UK, but I am an international student and am staying in my home country on summer break until the end of September.

Anyone know if it's still possible to successfully apply for Student Pack (and how)?

Many thanks!

**Comment by arafajs** [comment]
Yes, it is still possible to successfully apply for the GitHub Student Developer Pack even if you are an international student and are currently staying in your home country. However, you will need to provide additional documentation to verify your student status. This documentation may include:

- A copy of your student ID with your current enrollment date
- A class schedule
- A transcript
- An affiliation or enrollment verification letter from your school

**Reply by alvin-hong** [reply]
The approval website only allows uploads taken directly from the device camera. How are we able to upload something like a transcript that way? Also is it possible to get approved during the summer, because my current location is not at the school?

**Reply by samus-aran** [reply]
Hi @alvin-hong 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [You appear to not be near any campus location for the school you have selected.](https://github.com/orgs/community/discussions/111352#not-near-any-campus-location-for-selected-school)
- [The school you selected does not appear to have a campus location in your country.](https://github.com/orgs/community/discussions/111352#selected-school-does-not-appear-to-have-a-campus-location-in-your-country)
- Regarding to your vacation question I suspect you will need to leave the application until the start of the new semester to be on campus unless you are a remote student and can provide evidence as such.

Please don't revive old discussions as the information may not be relevant anymore. Open up a new discussion and we'll assist with your query. 👍
